integration-tests/src/test_loop/tests/resharding_v3.rs

```diff
@@ -1370,6 +1422,26 @@ fn test_resharding_v3_shard_shuffling() {
     test_resharding_v3_base(params);
 }
 
+#[test]
+fn test_resharding_v3_shard_shuffling_intense() {
+    let chunk_ranges_to_drop = HashMap::from([
+        (ShardUId { shard_id: 0, version: 3 }, -1..2),
+        (ShardUId { shard_id: 1, version: 3 }, -3..0),
+        (ShardUId { shard_id: 2, version: 3 }, -3..0),
+        (ShardUId { shard_id: 3, version: 3 }, 0..3),
+        (ShardUId { shard_id: 4, version: 3 }, 0..1),
+    ]);
+    let mut params = TestReshardingParameters::new()
+        .base_shard_layout_version(1)
```

Comment:
I would advice either:
* use the default
* make this test parametrized by the version and have one test for each